lib/src/iterable_extensions.dart

```diff
@@ -601,6 +601,19 @@ extension IterableExtension<T> on Iterable<T> {
       yield slice;
     }
   }
+
+  /// Returns a map where the keys are the unique elements of the iterable
+  /// and the values are the counts of those elements.
+  ///
+  /// For example, `['a', 'b', 'b', 'c', 'c', 'c'].countFrequency()`
+  /// returns `{'a': 1, 'b': 2, 'c': 3}`.
+  Map<T, int> countFrequency() {
```

Comment:
Would probably make it `get frequencies` instead of a function.

Reply:
thank you, I agree this way is better, please check my changes